src/lib/media/Voice.ts

```diff
@@ -194,6 +195,7 @@ export class CallRoom {
             let stream = await VoiceRTCInstance.getLocalStream()
             log.debug(`Sending local stream ${stream} to ${peer}`)
             room.addStream(stream, peer)
+            playSound(Sounds.Joined)
```

Comment:
i think this is fine because its when the user then is actually inside the room. accepting the call doesnt mean at that moment the user is actually in the call